src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -94,14 +94,17 @@ def start_formula(
     ) -> Formula[Quantity]:
         """Start execution of the given formula.
 
-        TODO: link to formula syntax.
+        Formulas can have Component IDs that are preceeded by a pound symbol("#"),
+        constant values and these operators: +, -, *, /, (, ).
```

Comment:
Maybe while you are improving this docstring, you can use a more consistent style for *literals*.

```suggestion
        Formulas can have Component IDs that are preceeded by a pound symbol(`#`),
        constant values and these operators: `+`, `-`, `*`, `/`, `(`, `)`.
```

Reply:
You didn't replied to this so pinging in case you missed it, but OK to leave it as it is, it is very minor.

Reply:
Oh, damn, I see it is done.